lms/djangoapps/certificates/management/commands/cert_generation.py

```diff
@@ -0,0 +1,75 @@
+"""
+Management command to generate course certificates for one or more users in a given course run.
+"""
+
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from lms.djangoapps.certificates.generation_handler import generate_certificate_task
+
+User = get_user_model()
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command to generate course certificates for one or more users in a given course run.
+
+    Example usage:
+    ./manage.py lms cert_generation -u 123 456 -c course-v1:edX+DemoX+Demo_Course
+    """
+
+    help = """
+    Generate course certificates for one or more users in a given course run.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-u', '--user',
+            nargs="+",
+            metavar='USER',
+            dest='user',
+            required=True,
+            help='user_id or space-separated list of user_ids for whom to generate course certificates'
+        )
+        parser.add_argument(
+            '-c', '--course-key',
+            metavar='COURSE_KEY',
+            dest='course_key',
+            required=True,
+            help="course run key"
+        )
+
+    def handle(self, *args, **options):
+        if not options["user"]:
```

Comment:
I would assume setting the parser argument to `required=True` would protect against this issue and make this code redundant. I am having trouble finding a good doc on the method though so feel free to leave it if you like.

Reply:
Nit: You use double quotes here and single quotes elsewhere.

Reply:
Thanks for the quotes callout. I'll fix that.

There are plans to come back and add a configuration model so that this command can be easily called from a jenkins job. At that point in time _required=True_ will be removed and these lines will be used (so I'm just planning for that). See the existing _cert_allowlist_generation.py_ command for reference.